tsdoc/etc/tsdoc.api.md

```diff
@@ -1151,6 +1151,10 @@ export class StandardTags {
     static readonly sealed: TSDocTagDefinition;
     static readonly see: TSDocTagDefinition;
     static readonly throws: TSDocTagDefinition;
+    static readonly tsBuiltInJsx: TSDocTagDefinition;
+    static readonly tsBuiltInJsxFrag: TSDocTagDefinition;
+    static readonly tsBuiltInJsxImportSource: TSDocTagDefinition;
+    static readonly tsBuiltInJsxRuntime: TSDocTagDefinition;
```

Comment:
Why not just call it `jsx`, `jsxFrag`, etc.?  

In my view, the fact that TypeScript originated these tags isn't that important.  Being "standard" means they should have the same meaning to every tool, not just `tsc`.